openapi/v2/Bom/bom-get.yml

```diff
@@ -0,0 +1,209 @@
+        For dev environments, provide the namespace to your dev org.
+paths:
+  '/item/{itemNumber}/bom':
+    get:
+      summary: BOM
+      description: Query to return a list of assembly records which represent the full BOM for the
+        given Item.
+      operationId: getBom
+      tags:
+      - Bill of Material
+      - BOM
+      - Assembly
+      parameters:
+      - name: itemNumber
+        in: path
+        description: Item Revision Id or Item Number of assembly to retrieve the BOM for. If Item Id or Item Number is provided, will default to latest released revision or latest revision if there is no released revision.
+        required: true
+        schema:
+          type: string
+          format: string
+      - name: level
+        in: query
+        description: Level of BOM to fetch. Defaults to All levels.
+        required: false
+        schema:
+          type: integer
+          format: integer
+          default: 0
+      - name: delta
+        in: query
+        description: Returns the deltas in the BOM between the latest released revision and previous revision in addition to the BOM.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      - name: flatten
+        in: query
+        description: Returns a list of parts in the assembly without level. Will combine duplicate BOM rows that exist at different levels into a single BOM row.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      responses:
+        '200':
+          description: Bill of Materials for a specified Item
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetBom'
+        '404':
+          description: Change Order not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+components:
+  schemas:
+    GetBom:
+      type: object
+      required:
+      - returnBom
+      properties:
+        returnBom:
+          type: array
+          items:
+            $ref: '#/components/schemas/ReturnBom'
+          description: Array of objects containing information about the manufacturer items and assemblies associated with the bom.
+        deltaBom:
+          type: object
+          items:
+            $ref: '#/components/schemas/DeltaBom'
+          description: Contains information about changes in the bom.
+    ReturnBom:
+      type: array
+      items:
+            $ref: '#/components/schemas/ReturnBomItem'
+    ReturnBomItem:
+      type: object
+      required:
+      - id
+      - attributes
+      properties:
+        Id:
+          type: string
+          description: Id of the associated object
+        attributes:
+          type: object
+          required:
+          - type
+          - url
+          properties:
```

Comment:
Where'd type and URL come from?

Reply:
"attributes": {
                "type": "PLMSM__Manufacturer_Item__c",
                "url": "/services/data/v49.0/sobjects/PLMSM__Manufacturer_Item__c/a0b1U00002oDxGTQA0"
            }